src/components/layout/modal/update_browser_alert_modal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react'
+import { ModalRefType } from 'components/elements/modal'
+import { Localize } from 'components/localization'
+import { Modal, Header } from 'components/elements'
+import { useUserBrowser } from 'components/hooks/use-user-browser'
+import { CookieStorage } from 'common/storage'
+
+const BrowserUpdateAlertModal = () => {
+    const modalRef = React.useRef<ModalRefType>()
+    const { is_outdated } = useUserBrowser()
+    const cookie_key = 'update_browser_alert_modal_shown'
+    const cookie = new CookieStorage(cookie_key)
+
+    useEffect(
+        () => is_outdated && !cookie.get(cookie_key) && modalRef.current.show(),
+        [is_outdated],
+    )
+
+    const handlePositive = () => {
+        cookie.set(cookie_key, true)
```

Comment:
Do we need any expiry date for the cookie?

Reply:
I have not pushed the new changes. today I'll add expiry date as well. what do you suggest for expiration time? is 14 days enough?

Reply:
Yeah, 14days sounds good to me